analyzer/tests/functional/analyze/test_analyze.py

```diff
@@ -181,22 +182,29 @@ def test_compiler_info_file_is_loaded(self):
             source.write(simple_file_content)
 
         with open(compiler_info_file, 'w') as source:
-            source.write('{"clang++": {"default_standard": "-std=FAKE_STD", '
-                         '"target": "FAKE_TARGET", "includes": ["-isystem '
-                         '/FAKE_INCLUDE_DIR"]}}')
+            source.write('''{
```

Comment:
Would we be able to handle existing compiler_info_files? Seems like the structure of the json is changed.
This is a problem for us, because we have existing report dirs with the old format in the CI jobs.
Either we can handle the old format, or could you please adapt https://github.com/Ericsson/codechecker/blob/master/scripts/debug_tools/renew_info_files.py to support this yet newer format?

Reply:
I've updated the renew_info_files script.